api/src/main/kotlin/com/integreety/activityengine/client/ActivityEngineClient.kt

```diff
@@ -0,0 +1,23 @@
+package com.integreety.activityengine.client
+
+import com.integreety.activityengine.api.request.ActivityRequest
+import com.integreety.activityengine.api.response.ActivityResponse
+import org.springframework.cloud.openfeign.FeignClient
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestParam
+
+@FeignClient(name = "activityEngineClient", url = "\${activityEngine.url}")
```

Comment:
Is this backslash for escaping the dollar sign?

Reply:
Correct